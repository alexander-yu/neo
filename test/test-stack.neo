void main(){

    create matrix<int> a = [[1,2],[3,4],[5,6]];
    create matrix<int> a1 = [[10,20,50,2],[30,40,40,1],[50,60,20,3]];
    println(ihstack(a, a1));

    create matrix<float> b = [[1.1,2.3],[3.1,4.0],[5.2,6.0]];
    create matrix<float> b1 = [[10.13,-20.5,50.1,2.1],[30.9,40.8,40.8,1.1],[50.1,60.2,20.4,3.0]];
    println(fhstack(b, b1));

    create matrix<int> c = [[1,2],[3,4],[5,6]];
    create matrix<int> c1 = [[10,20], [50,2], [30,40], [40,1], [50,60], [20,3]];
    println(ivstack(c, c1));

    create matrix<float> d = [[1.1,2.4,3.1],[4.3,5.0,6.1]];
    create matrix<float> d1 = [[10.1,20.3,50.2]];
    println(fvstack(d, d1));
}