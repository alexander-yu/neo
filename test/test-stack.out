[[1, 2, 10, 20, 50, 2],
 [3, 4, 30, 40, 40, 1],
 [5, 6, 50, 60, 20, 3]]
[[1.1, 2.3, 10.13, -20.5, 50.1, 2.1],
 [3.1, 4, 30.9, 40.8, 40.8, 1.1],
 [5.2, 6, 50.1, 60.2, 20.4, 3]]
[[1, 2],
 [3, 4],
 [5, 6],
 [10, 20],
 [50, 2],
 [30, 40],
 [40, 1],
 [50, 60],
 [20, 3]]
[[1.1, 2.4, 3.1],
 [4.3, 5, 6.1],
 [10.1, 20.3, 50.2]]